### DISCUSSION TITLE
0.8.0-beta.3版本官方Demo手机微信打开黑屏

### DISCUSSION THREAD

**Discussion by andyye136** [galacean/discussion#28] [Engine] [answered]
0.8.0-beta.3版本下的官方Demo在手机微信打开都是黑屏

https://oasisengine.cn/0.8/examples#animation-event
![image](https://user-images.githubusercontent.com/115980059/196160794-b2c3abf6-dced-406d-aeed-8778688aceff.png)

手机微信打开黑屏
![image](https://user-images.githubusercontent.com/115980059/196160960-4323066f-9dd1-4b66-96d2-18b4cecfc7d8.png)

**Comment by eyworldwide** [comment]
手机型号等信息反馈一下

**Comment by eyworldwide** [comment, marked as answer]
@andyye136 1.0 版本以下确实存在兼容问题，请升级到 1.0 或以上版本： https://galacean.antgroup.com/engine/examples/1.0/animation-event
